### PR TITLE
Keep the toolbar handle in bounds

### PR DESCRIPTION
**Pull request by matthiask**

### CODE REVIEW

File: debug_toolbar/static/debug_toolbar/js/toolbar.js [hunk 1]
```diff
@@ -210,23 +211,24 @@ const djdt = {
             e.classList.remove("djdt-active");
         });
     },
+    ensure_handle_visibility() {
+        const handle = document.getElementById("djDebugToolbarHandle");
+        // set handle position
+        const handleTop = Math.min(
+            localStorage.getItem("djdt.top") || 0,
+            window.innerHeight - handle.offsetWidth
```
Comment (tim-schilling) [inline debug_toolbar/static/debug_toolbar/js/toolbar.js:219]: Do we need to cover the case in which the `handle.offsetWidth` is greater than `window.innerHeight`?

Comment (matthiask) [reply debug_toolbar/static/debug_toolbar/js/toolbar.js:219]: Thanks! `offsetWidth` is about `74px` right now, I don't think we care for the case when the window height is less than a few hundred pixels.

Comment (tim-schilling) [reply debug_toolbar/static/debug_toolbar/js/toolbar.js:219]: Good stuff!

**Review by tim-schilling — APPROVED** [review]

**Comment by matthiask** [comment]
Thanks!

**Pull request merged**